public/account.html

```diff
@@ -83,7 +83,9 @@
 
           <p>
             <button class="primary" type="submit">Send</button>
-            <a href="#" data-action="show-signin">Cancel</a>
+            <small>
```

Comment:
@nickcolley  Gregor offered a suggestion to make a `.btn-link` class that is applied to these choices that are buttons that just look like a link. In the class we can add enough padding just like the buttons have or similar. How's that fix?